packages/serialize/types/index.d.ts

```diff
@@ -1,19 +1,26 @@
 // Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
 // TypeScript Version: 2.8
-
-import { RegisteredCache, SerializedStyles } from '@emotion/utils'
+import type { RegisteredCache } from '@emotion/cache'
 import * as CSS from 'csstype'
 
-export { RegisteredCache, SerializedStyles }
+export interface SerializedStyles {
+  name: string
+  styles: string
+  map?: string
+  next?: SerializedStyles
+}
 
 export type CSSProperties = CSS.PropertiesFallback<number | string>
 export type CSSPropertiesWithMultiValues = {
   [K in keyof CSSProperties]:
+    | null
+    | undefined
+    | false
```

Comment:
So I guess those were added to account for "a test that allows false value but not in TS"?

Reply:
Yes, but a weird thing happened, I managed to build my own serializer based on the implementation but using `js-beautify` like `beautify.css()` instead of using `@emotion/css-prettifier`.

When I used it and passed `null` like `css({ textAlign: null })`, it created a weird selector like `.emotion-0 textAlign {}` but when I switched to `@emotion/css-prettifier`, it removed it.

The `@emotion/css-prettifier` also removes the empty selector like `.emotion-0 {}` which I guess does the same thing hence why in the test we're not seeing these weird things.

It's not an issue as I can just use that lib you guys already have but just a weird thing that it only happens when it's `null`